env_data_folders/index.htm

```diff
@@ -95,10 +93,11 @@
   </nav>
 
   <div id="container" class="container">
-
+    <div class="page-header">
+      <h1>Main Controls <small>Basic LED strip control</small></h1>
```

Comment:
Header should be set dynamically, based on SectionFieldType - even if the fiorst section shall be always visible at the top.